modules/portalv2/js/src/main/scala/vinyldns/portalv2JS/components/LeftNav.scala

```
@@ -0,0 +1,66 @@
+package vinyldns.portalv2JS.components
+
+import scalacss.ScalaCssReact._
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.component.Scala.Unmounted
+import japgolly.scalajs.react.extra.Reusability
+import japgolly.scalajs.react.extra.router.RouterCtl
+import japgolly.scalajs.react.vdom.html_<^._
+import vinyldns.portalv2JS.models.Menu
+import vinyldns.portalv2JS.routes.AppRouter.AppPage
+
+object LeftNav {
+  val CssSettings = scalacss.devOrProdDefaults
```

looking at the ScalaCssReact docs you can import scalacss.ScalaCssReact.ProdDefaults, or scalacss.ScalaCssReact.DevDefaults, the prod one suppresses errors among other things. This dev or prod macro checks a jvm option and imports the dev settings by default